app/src/main/java/com/example/gradeassist/service/ScoreServiceStub.kt

```diff
@@ -1,11 +1,11 @@
 package com.example.gradeassist.service
 
+import androidx.lifecycle.MutableLiveData
 import com.example.gradeassist.dto.Score
 
 public class ScoreServiceStub : IScoreService {
 
-    override fun fetchById(scoreId: Int): Score {
-        return Score(52)
-    }
+    override fun fetchScoreById(scoreId: Int): MutableLiveData<Score> {
+        return MutableLiveData<Score>()   }
 
```

Comment:
fixed this stub to return mutableLiveData (Not really sure if necessary)

Reply:
If you don't use it, it is clutter and should be left out.  Clutter is technical debt.